Assets/HoloToolkit/Utilities/Scripts/HeadsetAdjustment.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using HoloToolkit.Unity.InputModule;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class HeadsetAdjustment : MonoBehaviour, IInputClickHandler, ISpeechHandler {
+
+    [Tooltip("The name of the scene to load when the user is ready. If left empty, the next scene is loaded as specified in the 'Scenes in Build')")]
+    public string NextSceneName;
+
+    public void OnInputClicked(InputClickedEventData eventData)
+    {
+        Debug.Log("Clicked");
+        GotoNextScene();
+    }
+
+    private void GotoNextScene()
+    {
+        if (!string.IsNullOrEmpty(NextSceneName))
+        {
+            SceneManager.LoadScene(NextSceneName);
+        }
+        else
+        {
+            var sceneIndex = SceneManager.GetActiveScene().buildIndex;
+            SceneManager.LoadScene(sceneIndex + 1);
```

Comment:
The documentation for `SceneManager.LoadScene` isn't clear about what happens if the provided `sceneBuildIndex` is out of range. Do we need to be defensive against the edge-case where the HeadsetAdjustment scene is the only scene in the build?